js/index.js

```diff
@@ -1,23 +1,38 @@
 // ITERATION 1
-
+ //... your code goes here
 function updateSubtotal(product) {
   console.log('Calculating subtotal, yey!');
 
-  //... your code goes here
+  const price = product.querySelector('.price span');
+  const quantity = product.querySelector('.quantity input');
+
+  const subtotal = price.innerHTML*(quantity.value);
+  product.querySelector('.subtotal span').innerHTML=subtotal;
+  return subtotal;
 }
 
 function calculateAll() {
   // code in the following two lines is added just for testing purposes.
   // it runs when only iteration 1 is completed. at later point, it can be removed.
-  const singleProduct = document.querySelector('.product');
-  updateSubtotal(singleProduct);
+  // const singleProduct = document.querySelector('.product');
+  // updateSubtotal(singleProduct);
   // end of test
 
   // ITERATION 2
   //... your code goes here
+  const multipleProducts = document.getElementsByClassName ('.product');
```

Comment:
```suggestion
  const multipleProducts = document.getElementsByClassName ('product');
```
Cuando utilices _getElementByClassName_ como ya sobreentiende que le vas a indicar una clase, no hace falta colocar el punto delante, por eso no te funcionaba la función de calcular el subtotal.